### ISSUE TITLE
Swagger UI does not display value in @ApiModelProperty when the property is a model

### ISSUE THREAD

**Issue by programus** [springfox/springfox#2237]
Please take the time to search the repository, if your question has already been asked or answered.

When I document my API, I put some description as the `value` of `@ApiModelProperty`, but the content disappears in the Swagger UI, if the property is an instance of another Model type with a `@ApiModel`. Only the description of the `@ApiModel` there. 
For example, I have two properties of the same type: 

```
	@ApiModelProperty(value = "the original plan...", required = true, allowEmptyValue = false, position = 0)
	public CreateRecommendMasterPlanOutDTO getOriginalPlan() {
		return originalPlan;
	}


	@ApiModelProperty(value = "the modified plan...", required = true, allowEmptyValue = false, position = 10)
	public CreateRecommendMasterPlanOutDTO getModifiedPlan() {
		return modifiedPlan;
	}
```

And I have the description of the type.
```
@ApiModel("Type description here")
public class CreateRecommendMasterPlanOutDTO extends BaseDTO {
//...
}
```

In the Swagger UI document, there is only the `Type description here`. I hope I can also get the descriptions of my properties or nobody understand what the difference between the two properties is. 

Is there anyway to get the descriptions of properties back?

**Comment by dilipkrish** [comment]
It should be available in the swagger json document.

**Comment by programus** [comment]
Yes, it is available in the json. However, I want to use swagger UI as the presentation of the document. Is there any advice?

**Comment by dilipkrish** [comment]
That would be a feature request for [Swagger-ui](https://github.com/swagger-api/swagger-ui)

**Comment by programus** [comment]
Thank you for your information. Is there any guide about how I can customize Swagger-ui provided by springfox-swagger-ui without building a new web project by myself?

**Comment by dilipkrish** [comment]
Not really unfortunately! Sorry

**Comment by programus** [comment]
So if I want to use a customized Swagger UI, I have to download a Swagger UI, customize it to use the URL of my document and then deploy to my server. 
The Swagger UI shipped with springfox will be not used, and I could even remove it from pom.xml. 
Am I understand it correctly?

**Comment by dilipkrish** [comment]
That is correct. The bundled swagger-ui is an opinionated 80-20 solution.
